lib/kubernetes-deploy/runner_task.rb

```diff
@@ -68,11 +68,8 @@ def validate_configuration(task_template, args)
       begin
         @kubeclient.get_namespace(@namespace) if @namespace.present?
       rescue KubeException => e
-        errors << if e.error_code == 404
-          "Namespace was not found"
-        else
-          "Could not connect to kubernetes cluster"
-        end
+        msg = e.error_code == 404 ? "Namespace was not found" : "Could not connect to kubernetes cluster"
```

Comment:
What was wrong with the old formulation? I find it easier to read than the ternary operator.